src/Stratis.Bitcoin.IntegrationTests/Miners/ProofOfStakeMintCoinsSpecification.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Stratis.Bitcoin.IntegrationTests.TestFramework;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Miners
+{
+    public partial class ProofOfStakeMintCoinsSpecification : BddSpecification
+    {
+        [Fact]
+        public void Staking_wallet_can_mint_new_coins()
+        {
+            Given(a_proof_of_work_node_with_wallet);
+            And(it_mines_genesis_and_premine_blocks);
```

Comment:
These two steps together feel like they could be their own GWT (Given When Then) in another scenario with a `Then 98 million coins are mined`.  The rest of this test could start from that point as a given.